coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 95.55
+    "covered_percent": 94.89
```

Comment:
Ehh, let's check your `coverage/index.html` to find some additional testing targets